wallaby.js

```diff
@@ -33,8 +33,8 @@ module.exports = wallaby => {
 
       // Tell Jest how to resolve symlinked modules. Without this, Jest will look at source TS files and not at Wallaby's compiled & instrumented files
       (jestConfig.moduleNameMapper = {
-        '@workday/canvas-kit-react-(.*)': '<rootDir>/modules/$1/react',
-        '@workday/canvas-kit-labs-react-(.*)': '<rootDir>/modules/_labs/$1/react',
+        '@workday/canvas-kit-react-([^/]+)(/?.*)': '<rootDir>/modules/$1/react$2',
+        '@workday/canvas-kit-labs-react-([^/]+)(/?.*)': '<rootDir>/modules/_labs/$1/react$2',
```

Comment:
This fixes WallabyJS module resolutions. Before we were getting mappings like this:
`@workday/canvas-kit-react-color-picker/lib/parts/ColorSwatch` -> `modules/color-picker/lib/parts/ColorSwatch/react`